spec/workspaces/slot.test.js

```diff
@@ -2,31 +2,35 @@ describe('Slot', function () {
   beforeEach(function () {
     var windowElement = jQuery('<div id="MOCK_WINDOW_1"/>');
     this.appendTo = jQuery('<div/>').append(windowElement);
-    this.eventEmitter = new Mirador.EventEmitter();
+    this.eventEmitter = new MockEventEmitter(new Mirador.EventEmitter());
 
     var mockWindow = {
       id: 'MOCK_WINDOW_1',
       element: windowElement
     };
 
-    var windowConfig = {
-      state: new Mirador.SaveController(jQuery.extend(true, {}, Mirador.DEFAULT_SETTINGS, {eventEmitter:this.eventEmitter}))
-    };
-
     this.slot = new Mirador.Slot({
       window: mockWindow,
       eventEmitter: this.eventEmitter
     });
   });
 
   describe('listenForActions', function () {
-    it('should respond to windowRemoved', function () {
```

Comment:
Yes, I think that should be added. Feel free to delete the test from here, but it should be added in Window.

If we can boost the test coverage a bit, I think this will be ready to go.